include/Methode.tex

```diff
@@ -53,5 +53,12 @@ \subsubsection{Anforderungen an Lösungsansätze, Proof of Concept und MVP}
 \end{itemize}
 
 \subsection{Einschränkungen und Abgrenzungen}
+In diesem Projekt wurde explizit auf eine Implementation mit dem Lagerverwaltungssystem verzichtet, da der Fokus auf der Machbarkeitsstudie lag. Es wurde diese Implementation zwar berücksichtigt, aber sie stellt eine Unbekannte für das Folgeprojekt dar. Weiter wurde explizit die Entwicklung des Prototypen zur Produktionsreife als nicht Teil des Projektes verstanden. Die Referenzimplementation wurde als Validierung der Konzepte und Versuche verstanden, und nicht als Prototyp zur Einbindung in die Umgebung.
```

Comment:
_Es wurde diese Implementation zwar berücksichtigt, aber sie stellt eine Unbekannte für das Folgeprojekt dar._ 
Was meinst du genau?
* Was wurde berücksichtigt
* Wie wurde es berücksichtigt
* Bei welcher Implementation wurde es berücksichtigt

Weiter wurde explizit die Entwicklung des **Prototypen** zur Produktionsreife als nicht Teil des Projektes verstanden. Die Referenzimplementation wurde als Validierung der Konzepte und Versuche verstanden, und nicht als **Prototyp** zur Einbindung in die Umgebung.
Wo gibt es dann ein Prototypen? -> im ersten Satz gibt es ja einen im Zweiten wird verständlich gemacht, dass es sich um keinen handelt.

Reply:
Wurde im Rahmen der Machbarkeitsstudie berücksichtig, sprich für das Folgeprojekt geplant